src/WebJobs.Script/Workers/Profiles/WorkerProfileManager.cs

```diff
@@ -42,6 +40,7 @@ private bool GetEvaluatedProfile(string language, out WorkerDescriptionProfile e
                 {
                     if (profile.EvaluateConditions())
                     {
+                        _logger.LogInformation($"Conditions evaluated succesfully for profile with name: {profile.Name} and ID: {profile.ProfileId}");
```

Comment:
For all logging here, please pass the values as arguments into the log methods instead of using interpolation/concatenating the strings.